### PR TITLE
osdeps: use 1.7.x repository for bionic

### PR DESCRIPTION
**Pull request by scollazo**
The repository for 1.8.x shouldn't be use in 1.7.2.

This connects to archivematica/Issues#119.

### CODE REVIEW

File: src/MCPClient/osdeps/Ubuntu-18.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   ],
   "osdeps_repos": [
     "deb [arch=amd64] http://packages.archivematica.org/1.7.x/ubuntu-externals xenial main",
```
Comment (sevein) [inline src/MCPClient/osdeps/Ubuntu-18.json:13]: Why is this needed?

Comment (scollazo) [reply src/MCPClient/osdeps/Ubuntu-18.json:13]: Some dependencies have not been rebuilt for 18.04, as the packages from 16.04 work without problems. I would prefer to leave 1.7 as is, and improve the repository management for 1.8

**Review by sevein — APPROVED** [review]
